features/old/applications/buyers/extra_fields.feature

```diff
@@ -23,7 +23,6 @@ Feature: Buyer side, application extra fields
     And I go to the new application page
     Then I should see the fields:
       | name        |
-      | Description |
```

Comment:
After adding `optional_fields_are :description` these changes should not be needed. Could you please revert and check?

Reply:
it still fails and after this it works fine

Reply:
That's true. The reason is that when a new provider is created, Fields Definitions are created only based on the fields that are in the `required_fields_are`:
https://github.com/3scale/porta/blob/8aa9ee90fdb36b1f7f69d51a8276c7f6383adfd1/app/models/fields_definition.rb#L59-L62

Reply:
Yes, by default it's not added to the list of fields for Application, but the providers can add this field and make it either required or not.